Not as easy as I thought. Going to skip the update.

``` 
unpacking sources
unpacking source archive /nix/store/djvqnjj2faisyk9q7xc75c9apswjn9pz-source
source root is source
patching sources
applying patch /nix/store/06hyzsh6v7gdpx5b88zqrr0h4sx0hlm9-Cargo.toml.patch
patching file Cargo.lock
patching file Cargo.toml
configuring
no configure script, doing nothing
building
warning: patch for `async-std` uses the features mechanism. default-features and features will not take effect because the patch dependency does not support this mechanism
    Updating git repository `https://github.com/async-rs/async-std`
    Updating crates.io index
    Updating git repository `https://github.com/piegamesde/cli-clipboard`
error: failed to sync

Caused by:
  failed to load pkg lockfile

Caused by:
  failed to get `cli-clipboard` as a dependency of package `wormhole-rs v0.4.0 (/build/source/cli)`

Caused by:
  failed to load source for dependency `cli-clipboard`

Caused by:
  Unable to update https://github.com/piegamesde/cli-clipboard#8447bd4a

Caused by:
  object not found - no match for id (8447bd4ac00af578d21ff440752e7aa837c07f30); class=Odb (9); code=NotFound (-3)
Traceback (most recent call last):
  File "/nix/store/r3nwnppm7n0fbnynfrx2q0dzjz5dgfar-cargo-vendor-normalise/bin/.cargo-vendor-normalise-wrapped", line 42, in <module>
    main()
  File "/nix/store/r3nwnppm7n0fbnynfrx2q0dzjz5dgfar-cargo-vendor-normalise/bin/.cargo-vendor-normalise-wrapped", line 17, in main
    assert list(data.keys()) == ["source"]
AssertionError
```

--- a/pkgs/tools/networking/magic-wormhole-rs/default.nix
+++ b/pkgs/tools/networking/magic-wormhole-rs/default.nix
@@ -4,7 +4,7 @@
 , rustPlatform
 }:
 rustPlatform.buildRustPackage rec {
-  name = "magic-wormhole-rs";
+  pname = "magic-wormhole-rs";
   version = "0.3.0";
 
   src = fetchFromGitHub {
@@ -18,7 +18,7 @@ rustPlatform.buildRustPackage rec {
   # https://github.com/NixOS/nixpkgs/issues/30742
   # and can probably be removed once the issue is resolved
   cargoPatches = [ ./Cargo.toml.patch ];
-  cargoSha256 = "sha256-DG1kyukgzDbolX9Mg9hK1TRyzIWbAX6f54jSM8clj/c=";
+  cargoSha256 = "sha256-ujwvwr4GR/rQWnXFfL8sqPyz4QvGeOxwBrT+gf+vjsI=";
 
   # all tests involve networking and are bound fail
   doCheck = false;

